content/docs/studio/get-started.md

```diff
@@ -105,6 +107,13 @@ You should now see that a project has been added in your dashboard.
 
 2. You can also make the project public by [sharing it on the web].
 
+## Submit experiments and track live metrics and plots
```

Comment:
💅🏼 but this is going to be long in the CONTENT bar:

<img width="758" alt="image" src="https://user-images.githubusercontent.com/1477535/199326844-91dc9da7-63c3-4081-930e-fce8ff54cc83.png">

> Try to always sanity check changes on the review app, in this case https://dvc-org-live-metrics-bw8woadv9.herokuapp.com/doc/studio/get-started.